plasmapy/dispersion/numerical/tests/test_hollweg_.py

```diff
@@ -256,6 +256,14 @@ def test_z_mean_override(self, kwargs, expected):
                 },
                 {"shape": (3,)},
             ),
+            (
+                {
+                    **_kwargs_single_valued,
+                    "k": [1, 2, 3] * u.rad / u.m,
+                    "theta": [50, 77] * u.deg,
+                },
+                {"shape": (3, 2)},
+            ),
```

Comment:
Should also add a test case where `k` is single valued and `theta` is an array.

Reply:
Added 👍